packages/core/test/scenarios/transforms.js

```diff
@@ -46,6 +48,7 @@ module.exports = [
       name: 'transforms - common pattern "_inheritsLoose" works across package boundaries',
       defineOne: () => {
         /* eslint-disable */
+        // @ts-ignore
```

Comment:
which errors are we suppressing btw?
i spotted no errors removing our 3 `// @ts-ignore` comments
and our types still rebuild fine

Reply:
probably leftover from when I turned on type checks for all JS files and then turned it off after I realized what I'd be in for.